scheduler/src/cook/mesos/mesos_compute_cluster.clj

```diff
@@ -24,17 +24,56 @@
             [cook.mesos.task :as task]
             [cook.plugins.definitions :as plugins]
             [cook.plugins.pool :as pool-plugin]
+            [cook.pool :as pool]
             [cook.progress :as progress]
             [cook.scheduler.scheduler :as sched]
+            [cook.tools :as tools]
             [datomic.api :as d]
             [mesomatic.scheduler :as mesos]
             [metrics.meters :as meters]
-            [plumbing.core :as pc]))
+            [plumbing.core :as pc]
+            [metrics.counters :as counters]))
 
 (meters/defmeter [cook-mesos scheduler mesos-error])
 (meters/defmeter [cook-mesos scheduler handle-framework-message-rate])
 (meters/defmeter [cook-mesos scheduler handle-status-update-rate])
+(counters/defcounter [cook-mesos scheduler offer-chan-depth])
 
+(def offer-chan-size 100)
```

Comment:
What happens if we overflow this. I'm assumign this is no change from prior behavior?

Reply:
We just log errors. This was copied out of scheduler.clj